src/orb-utils/orb.yml

```diff
@@ -0,0 +1,10 @@
+version: 2.1
+commands:
+  install-circleci-cli:
+    steps:
+      - run:
+          name: "Install `circleci` CLI"
+          command: |
+            sudo bash -c "$(curl -fSl https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh)"
```

Comment:
Do we want to do some kind of a integrity check in that install script? (Not really part of this PR)

Reply:
Like using a checksum?

Reply:
sha256 would work well